python/ray/util/client/__init__.py

```diff
@@ -73,7 +74,8 @@ def connect(self,
                 conn_str,
                 secure=secure,
                 metadata=metadata,
-                connection_retries=connection_retries)
+                connection_retries=connection_retries,
+                reconnect_grace_period=reconnect_grace_period)
             self.api.worker = self.client_worker
```

Comment:
type annotation

Reply:
No longer relevant since grace period is only through env now